Gemfile

```diff
@@ -35,7 +35,7 @@ end
 if facterversion = ENV['FACTER_GEM_VERSION']
   gem 'facter', facterversion, :require => false
 else
-  gem 'facter', :require => false
+  gem 'facter', '~> 2.4.0', :require => false
```

Comment:
Generally I like a note why it's pinned. I'm pretty sure that in this case it's because the facterdb has no facts for 2.5 yet, but when going over it you either want a note in git blame or a comment in the file.